### DISCUSSION TITLE
Configuration Intellij Idea

### DISCUSSION THREAD

**Discussion by scamassa** [heig-vd-dai-course/heig-vd-dai-course#452] [Q&A]
Bonjour @ludelafo 
J'ai désinstallé et réinstallé Intellij Idea suite aux problèmes que j'ai eu il y a deux semaines.
J'ai ensuite eu des problèmes avec l'application de sécurité windows que j'ai du reset.
Maintenant j'essaie de refaire la configuration de Intellij Idea indiquée dans le cours, et j'ai cette erreur quand j'essaie de lancer la commande Net-NewFirewallRule. Quelqu'un aurait une idée du problème ?
![image](https://github.com/user-attachments/assets/5aa59d5c-0ac8-45c8-8972-02ca733759fd)

**Comment by ludelafo** [comment]
Bonjour @scamassa,

Juste pour être sûr, est-ce que vous avez bien lancé la commande en tant qu'Administrateur ?

**Reply by scamassa** [reply]
oui j'ai bien ouvert powershell en administrateur

**Reply by ludelafo** [reply]
Si vous ignorez cette erreur, est-ce que la suite de la procédure marche ?

Je n'ai jamais vu cette erreur et je pense que je devrais investiguer de vive voix. Est-ce que vous seriez dispo pour un call Teams demain vers 11h pour que l'on regarde ensemble ? Ou dès 16h30 si cela vous arrange mieux (selon votre horaire que je vois sur GAPS) ?

**Reply by scamassa** [reply]
je vais essayer de continuer la configuration et voir si j'ai des problèmes plus tard.
Oui je suis disponible à 11h demain !

**Reply by scamassa** [reply]
bonjour,
j'ai terminé la configuration et les problèmes ont l'air réglés pour l'instant, je peux de nouveau build et run depuis intellij et les projets maven sont reconnus.
J'espère que l'erreur du firewall ne me causera pas de problèmes (pour l'instant je ne peux de nouveau plus ouvrir Sécurité Windows, mais intellij fonctionne donc j'ai plus de temps pour régler le problème)

**Reply by ludelafo** [reply]
Super, je suis content de l'apprendre !

Au vu de ce que j'ai trouvé sur Internet avec votre erreur 0x80070057 et des problèmes liés à Windows Security, j'ai l'impression que votre système d'exploitation est corrompu/devrait être réinstallé pour repartir sur des bases clean.

Est-ce que vous avez pu valider que l'état de votre SSD/disque dur est en bon état ?
